gridstatus/pjm.py

```diff
@@ -941,6 +941,64 @@ def extract_short_name(row):
 
         return data
 
+    @support_date_range(frequency=None)
+    def get_it_sced_lmp_5_min(self, date, end=None, verbose=False):
+        """Get 5 minute LMPs from the Integrated Forward Market (IFM)"""
+
+        if date == "latest":
+            return self.get_it_sced_lmp_5_min("today", verbose=verbose)
+
+        params = {
+            "fields": (
+                "case_approval_datetime_utc,datetime_beginning_utc,itsced_lmp,marginal_congestion,marginal_loss,pnode_id,pnode_name"  # noqa: E501
+            ),
+        }
+
+        data = self._get_pjm_json(
+            "five_min_itsced_lmps",
+            start=date,
+            end=end,
+            params=params,
+            verbose=verbose,
+            interval_duration_min=5,
+        )
+
+        # TODO: are there other columns we want to rename?
+        data = data.rename(
+            columns={
+                "case_approval_datetime_utc": "Case Approval Time",
+            },
+        )
+
+        data.columns = data.columns.map(lambda x: x.replace("_", " ").title())
+
+        # LMP = Energy + Congestion + Loss so Energy = LMP - Congestion - Loss
+        data["Marginal Energy"] = (
+            data["Itsced Lmp"] - data["Marginal Congestion"] - data["Marginal Loss"]
+        )
+
+        data = data[
+            [
+                "Interval Start",
+                "Interval End",
+                "Case Approval Time",
+                "Pnode Id",
+                "Pnode Name",
+                "Itsced Lmp",
+                "Marginal Energy",
+                "Marginal Congestion",
+                "Marginal Loss",
```

Comment:
Is "Marginal" in these names redundant? If "LMP" is Locational Marginal Price then are the energy, congestion, and loss components assumed to be marginal?

Reply:
in other places, we do get rid of the `marginal`, i think we should do that here too